service/dap/server_test.go

```diff
@@ -4703,7 +4686,12 @@ func TestBadLaunchRequests(t *testing.T) {
 		}
 		checkFailedToLaunch(client.ExpectInvisibleErrorResponse(t))
 
-		client.LaunchRequest("" /*debug by default*/, fixture.Path+"_does_not_exist", stopOnEntry)
```

Comment:
doesn't just changing "" to "debug" in LaunchRequest() work without doing the entire args map?

Reply:
I reverted the change that rejects an empty mode, so reverted this test change too. Obsolete